runtime/src/lib.rs

```diff
@@ -244,10 +242,6 @@ impl frame_system::Config for Runtime {
     type Index = Index;
     /// The lookup mechanism to get account ID from whatever is passed in dispatchers.
     type Lookup = IdentityLookup<AccountId>;
-    /// Maximum size of all encoded transactions (in bytes) that are allowed in one block.
-    // type MaximumBlockLength = MaximumBlockLength;
-    /// Maximum weight of each block.
-    // type MaximumBlockWeight = MaximumBlockWeight;
     /// The maximum weight that a single extrinsic of `Normal` dispatch class can have,
     /// idependent of the logic of that extrinsics. (Roughly max block weight - average on
     /// initialize cost).
```

Comment:
why isn't `SystemWeightInfo` value `frame_system::weights::SubstrateWeight<Runtime>` like in standalone chain https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR301

Reply:
same as above. @ltfschoen I am not sure. I haven't changed this. It was already there https://github.com/DataHighway-DHX/node/blob/ilya/parachain-update/runtime/src/lib.rs#L268 in existing parachain code.

Can you create a ticket and then we can discuss with @festelo and @cgroeschel for the same?

Reply:
added to #227